rancher2/00_provider_test.go

```diff
@@ -35,7 +35,7 @@ func init() {
 	testAccRancher2AdminPass = testAccRancher2DefaultAdminPass
 	err := testAccCheck()
 	if err != nil {
-		log.Fatalf("%v", err)
+		log.Fatalf("failed check %s", err)
```

Comment:
wdyt of using `%w` instead o `%s`?

Reply:
That won't work?

https://go.dev/play/p/kOQqlRqoAIM

Reply:
ops, sorry, my mistake